index.html

```diff
@@ -1421,7 +1429,57 @@ <h3 id="applying">
           <h3>
             Updating the manifest
           </h3>
-          <aside class="issue" data-number="446"></aside>
+          <p>
+            As specified for [^link/rel/manifest^] link relation, the manifest
+            is fetched and processed on every page load. When the [=processing
+            a manifest=] is successful, user agents MAY apply updated manifest
+            to any current and future <a>application contexts</a> associated
+            with the application.
+          </p>
+          <p>
+            For the purpose of updating, the following member are
+            <dfn>security-sensitive members</dfn>, as they are presented during
+            installation and on launch surfaces:
+          </p>
+          <ol>
+            <li>[=manifest/short_name=],
+            </li>
+            <li>[=manifest/icons=]
+            </li>
+            <li>[=manifest/name=],
+            </li>
+          </ol>
+          <p data-cite="permissions">
+            User agents SHOULD NOT automatically apply changes to
+            [=security-sensitive members=] without [=express permission=] from
+            the user.
+          </p>
+          <p>
+            Instead, user agents SHOULD present changes to [=security-sensitive
+            members=] with appropriate management options, so the user can make
+            an informed decision about updating the web application.
+          </p>
+          <p>
+            The user agent MAY automatically apply the changes if the update
+            does not contain changes to [=security-sensitive members=].
+          </p>
+          <aside class="note" title=
+          "A user agent will not apply a partial update">
+            <p>
+              When the update contains changes both to [=security-sensitive
```

Comment:
@marcoscaceres I've added some  context to the partial update note, let me know what you think.